tripal_chado/includes/tripal_chado.publish.inc

```diff
@@ -90,10 +90,22 @@ function tripal_chado_publish_form($form, &$form_state) {
 
         // Now call the widget callback function to let the module adjust the
         // element as needed.
+        if (!isset($instance['widget']['module'](( {
```

Comment:
```suggestion
        if (!isset($instance['widget']['module'])) {
```

Reply:
@bradfordcondon you should try again with this fix

Reply:
yep agreed